contributors.md

```diff
@@ -29,4 +29,9 @@
 - Place: Ahmedabad, Gujarat, India
 - Bio: Human with a desire for development
 - GitHub: [aditmodhvadia](https://github.com/aditmodhvadia/)
+
+### Name: [Siddhant Sharma](https://github.com/ssiddhantsharma/)
+- Place: Surat, Gujarat, India
+- Bio: A not-so-simple high school student!
+- GitHub: [ssiddhantsharma](https://github.com/ssiddhantsharma/)
 >>>>>>> 296d80d0bc3e6ec9e9f8869158a9e9dfa211a99c
```

Comment:
```suggestion

```
This line seems to be from  someone who tried to fix a merge conflict. Please remove it!